examples/tv-app/linux/include/application-launcher/ApplicationLauncherManager.cpp

```diff
@@ -17,12 +17,12 @@
  */
 
 #include "ApplicationLauncherManager.h"
-
+#include <app/Command.h>
```

Comment:
Why is Command.h needed here?

Reply:
Leftover from moving the code. Removed.

Reply:
Removed also more header files which are not needed now.